### PR TITLE
Password recovery changes

### PR DESCRIPTION
**Pull request by prajakta-kulkarni-git**
User should be able to get recovery token if clicked on password forgotten button.

### CODE REVIEW

File: client/src/Pages/password/login.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import { Form, Button } from "semantic-ui-react";
```
Comment (LiamANeeson) [inline client/src/Pages/password/login.js:1]: Can this be done utilising bootstrap so  we only are using one CSS Framework

File: client/src/Pages/password/login.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    }
+  }
+
+  const signInSchema = yup.object({
+    email: yup.string().required("Email/Username is required"),
+    password: yup.string().required("Password cannot be empty"),
+  });
+
+  const { handleChange, handleSubmit, handleBlur, errors, touched, values } =
+    useFormik({
```
Comment (LiamANeeson) [inline client/src/Pages/password/login.js:39]: Never heard of form milk before what does it do for us here?

**Review by LiamANeeson — COMMENTED** [review]
The code for password recovery looks good however not sure if we should be utilising multiple CSS frameworks as it could get messy.

File: client/src/Pages/password/login.js (hunk 2 above)
Comment (prajakta-kulkarni-git) [reply client/src/Pages/password/login.js]: I removed it though.

File: client/src/Pages/password/login.js (hunk 1 above)
Comment (prajakta-kulkarni-git) [reply client/src/Pages/password/login.js]: yes added bootstrap now

**Review by TejasJD — APPROVED** [review]
Good work

**Pull request merged**